### PR TITLE
Updating Docs for Custom Admin Roles for IDPs

### PR DESCRIPTION
**Pull request by kamaldeepcheema-okta**
## Description:
Updating the Public Api documents in the sections below to include relevant changes for adding Custom Admin Roles for Identity Providers:

[Administrator Roles | Okta Developer](https://developer.okta.com/docs/reference/api/roles/#permission-properties)

[Role Assignment | Okta Developer](https://developer.okta.com/docs/concepts/role-assignment/#resource-identifiers)

Is this PR related to a Monolith release? Yes. 2023.06.2

### Resolves:

* [OKTA-615157](https://oktainc.atlassian.net/browse/OKTA-615157)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md](https://github.com/okta/okta-developer-docs/blob/5585fc488cce75f1d6d4d800ad90abd548c74e9e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Frole-assignment%2Findex.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2409c8b7-3264-4d2b-bcd5-61a8c36419fe) | 77  | 84  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/roles/index.md](https://github.com/okta/okta-developer-docs/blob/5585fc488cce75f1d6d4d800ad90abd548c74e9e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Froles%2Findex.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/36796f9d-b946-4523-93c8-188128ab0e32) | 92  | 79  | 71  | 74  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.d8e1b8a7-c0f7-490c-922c-f58e17df413e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 1]
```diff
@@ -238,6 +238,8 @@ Standard resource sets are managed by Okta only and can't be updated or deleted.
 | `okta.authzServers.manage`                                  | Allows the admin to manage authorization servers                                                                                                    | All authorization servers, a specific authorization server |
 | `okta.customizations.read`                                  | Allows the admin to read customizations                                                                                                             | All customizations |
 | `okta.customizations.manage`                                | Allows the admin to manage customizations                                                                                                           | All customizations |
+| `okta.identityProviders.read`                               | Allows the admin to read identity providers                                                                                                           | All identity providers |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:241]:
```suggestion
| `okta.identityProviders.read` <ApiLifecycle access="ea" />  | Allows the admin to read Identity Providers                                                                                                           | All Identity Providers |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 2]
```diff
@@ -238,6 +238,8 @@ Standard resource sets are managed by Okta only and can't be updated or deleted.
...
+| `okta.identityProviders.manage`                             | Allows the admin to manage identity providers                                                                                                           | All identity providers |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:242]:
```suggestion
| `okta.identityProviders.manage` <ApiLifecycle access="ea" />  | Allows the admin to manage Identity Providers                                                                                                           | All Identity Providers |
```

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 3]
```diff
@@ -3463,6 +3463,8 @@ User permissions are only effective regarding the Groups to which the admin is g
 | `okta.authzServers.manage`              | Allows the admin to manage authorization servers                                                                                                    | All authorization servers, a specific authorization server |
 | `okta.customizations.read`              | Allows the admin to read customizations                                                                                                             | All customizations |
 | `okta.customizations.manage`            | Allows the admin to manage customizations                                                                                                           | All customizations |
+| `okta.identityProviders.read`           | Allows the admin to read identity providers                                                                                                           | All identity providers |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3466]:
```suggestion
| `okta.identityProviders.read` <ApiLifecycle access="ea" />  | Allows the admin to read Identity Providers                                                                                                           | All Identity Providers |
```

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 4]
```diff
@@ -3463,6 +3463,8 @@ User permissions are only effective regarding the Groups to which the admin is g
...
+| `okta.identityProviders.manage`         | Allows the admin to manage identity providers                                                                                                           | All identity providers |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3467]:
```suggestion
| `okta.identityProviders.manage`  <ApiLifecycle access="ea" />  | Allows the admin to manage Identity Providers                                                                                                           | All Identity Providers |
```

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 5]
```diff
@@ -3520,6 +3522,7 @@ The following are the supported resources.
...
 |                         | All Users within a specific Group                                   | `orn:${partition}:directory:${yourOrgId}:groups:${groupId}:contained_resources`       | [`https://${yourOktaDomain}/api/v1/groups/${groupId}/users`](/docs/reference/api/groups/#list-group-members)                                            |
 | Identity Provider       | All Apps                                                            | `orn:${partition}:idp:${yourOrgId}:apps`                                              | [`https://${yourOktaDomain}/api/v1/apps`](/docs/reference/api/apps/#list-applications)                                                                  |
+|                         | All Identity Providers                                              | `orn:${partition}:idp:${yourOrgId}:identity_provider`                                              | [`https://${yourOktaDomain}/api/v1/idps`](/docs/reference/api/idps/#list-identity-providers)                                                                  |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3525]:
```suggestion
|                         | All Identity Providers    <ApiLifecycle access="ea" />    | `orn:${partition}:idp:${yourOrgId}:identity_provider`  | [`https://${yourOktaDomain}/api/v1/idps`](/docs/reference/api/idps/#list-identity-providers)                                                                  |
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md](https://github.com/okta/okta-developer-docs/blob/3d8e2dc583e6ebc172fb8c8034cc0072585d388d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Frole-assignment%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94b5eadc-6fb3-40de-afd7-138d5cd211a9) | 87  | 84  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/roles/index.md](https://github.com/okta/okta-developer-docs/blob/3d8e2dc583e6ebc172fb8c8034cc0072585d388d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Froles%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/881c343f-dede-4e59-8938-8cf1363c73f6) | 96  | 79  | 71  | 74  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9205b3db-7756-4ae3-8b02-b6f70c5f7e94?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**